cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/resource/AbstractReadyResourceOperator.java

```diff
@@ -91,7 +91,8 @@ public void handle(Long timerId) {
                             long timeLeft = deadline - System.currentTimeMillis();
                             if (timeLeft <= 0) {
                                 log.error("Exceeded timeoutMs of {} ms while waiting for {} {} in namespace {} to be ready", timeoutMs, resourceKind, name, namespace);
-                                fut.fail(new TimeoutException());
+                                String exceptionMessage = String.format("Exceeded timeoutMs of %d ms while waiting for %s %s in namespace %s to be ready", timeoutMs, resourceKind, name, namespace);
```

Comment:
`Exceeded timeout of %dms` (the name of the local var `timeoutMs` doesn't add anything, and a duration is better written 100ms rather than 100 ms)